Ginger/GingerCoreNET/Run/RunSetActions/RunSetActionJSONSummary.cs

```diff
@@ -41,8 +41,16 @@ public override void Execute(ReportInfo RI)
         {
             string json = WorkSpace.Instance.RunsetExecutor.CreateSummary();
             string timestamp = DateTime.Now.ToString("MMddyyyy_HHmmss");
-            string testArtifactsFolder = WorkSpace.Instance.TestArtifactsFolder;            
-            string fileName = Path.Combine(testArtifactsFolder, WorkSpace.Instance.RunsetExecutor.RunSetConfig.Name + "_" + timestamp + ".json.txt");//why not as .json?                
+
+            string jsonSummaryFolder = WorkSpace.Instance.SolutionRepository.ConvertSolutionRelativePath(WorkSpace.Instance.Solution.LoggerConfigurations.CalculatedLoggerFolder);
+            if (amdocs.ginger.GingerCoreNET.WorkSpace.Instance.RunningInExecutionMode)
+            {
```

Comment:
if user didn't provided artifact path over CLI so WorkSpace.Instance.TestArtifactsFolder will be null? if yes so no need to validate if we RunningInExecutionMode, only check if WorkSpace.Instance.TestArtifactsFolder is not null